### PR TITLE
Clarify usage of timezones

### PR DESCRIPTION
**Pull request by fmueller**
This pull request includes change request by @tfrauenstein proposed in #4

### CODE REVIEW

File: data-formats/DataFormats.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ JSON-encode the body payload. The JSON payload must follow RFC-7158 and be compa
 
 * In HTTP headers (including the proprietary headers), use the [HTTP date format defined in RFC 7231](http://tools.ietf.org/html/rfc7231#section-7.1.1.1).
 
-Timestamps are passed in UTC-related format via APIs and should be stored in UTC (i.e. without any offset information). Localization based on UTC should be done by the services that provide user interfaces, if required.
+A zone offset may be used (both, in request and responses) -- this is simply defined by the standards. However, we encourage to restrict to UTC (and go without offsets) when dates are passed and returned. From experience we know that zone offsets are not easy to understand and often not correctly handled. Note also, that zone offsets are different from local times that may include summer time savings.
```
Comment (hjacobs) [inline data-formats/DataFormats.md]: "summer time saving" -> "daylight saving time" (summer time savings sounds "wrong")

Comment (fmueller) [inline data-formats/DataFormats.md]: Good point, fixed.

**Comment by fmueller** [comment]
Feedback for this change by you guys? @valgog @tfrauenstein @whiskeysierra @ePaul

**Comment by ePaul** [comment]
Looks okay after the fixed wording. :+1:

**Pull request merged**